crypto/monero/crypto.go

```diff
@@ -57,6 +57,22 @@ func NewPrivateKeyPairFromBytes(skBytes, vkBytes []byte) (*PrivateKeyPair, error
 	}, nil
 }
 
+// NewPrivateKeyPairFromHex returns a PrivateKeyPair from the given hex-encoded byte
+// representation of a private spend and view key.
+func NewPrivateKeyPairFromHex(skHex, vkHex string) (*PrivateKeyPair, error) {
```

Comment:
Why are we passing the view key when it is [derived from the spend key](https://github.com/AthanorLabs/atomic-swap/blob/master/crypto/monero/crypto.go#L137-L161)?

If we wanted it as a sanity check of the spend key, it could make sense, but I don't see the code doing this. Can we simplify? We only need to store the private spend key to the database.

Reply:
updated to only store spend keys in the db!